src/Shouldly/ShouldStaticClasses/ShouldThrowExtensionsOverloads.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using JetBrains.Annotations;
+
+namespace Shouldly
+{
+    public static partial class Should
```

Comment:
Im heading to bed (back in Australia now). If you don't make progress let me know, I will merge then have a look at solving in the morning when I get up